app/src/main/java/ch/epfl/sdp/Map/MapFragment.java

```diff
@@ -51,7 +51,7 @@
     private static final int MIN_UP_INTERVAL_MILLISECS = 1000;
     private static final int MIN_UP_INTERVAL_METERS = 5;
     private MapView mapView;
-    private MapboxMap map;
+    public MapboxMap map; // public for testing
```

Comment:
It would be better to use a private getter method with the "visibleForTesting" annotation